development/base/bot.py

```diff
@@ -0,0 +1,66 @@
+import base.settings as sett
+
+class Bot:
+    
+    def echo(self, message):
+        self.message = message
+        
+        return self.message
+    
+    def dialog(self, message):
+        self.message = message
+    
+        # preparing a message
+        message = message.lower()
+
+        # removing punctuation marks
+        message = message.replace('.', '').replace(',', '').replace('?', '').replace('!', '').replace('.', '').replace(':', '').replace(' ', '')
+
+
+        # data for dialog: recognizing meaning from messages
+        
+        greeting_message = ['привет', 'доброеутро', 'добрыйдень', 'добрыйвечер', 'здравствуй', 'здравствуйте', 'приветствую', 'прив', 'здрасти', 'ктоты', 'тыкто']
+
+        mood_message = ['какдела', 'каконо', 'какнастроение', 'какуспехи', 'всёнормально', 'тыкак', 'какты']
+
+        farewell_message = ['пока', 'прощай', 'до свидания', 'удачи', 'успехов', 'покеда', 'покедова']
+
+        isuct_message = ['химтех', 'хим', 'ивановскийгосударственныйхимикотехнологическийуниверситет', 'исукт', 'исакт', 'игхту', 'тылюбишьхимтех', 'тылюбишьхим', 'тебенравитсяхим', 'тебенравитсяхимтех', 'ахимтехнравится']
+
+        compliment_message = ['тыкрутой', 'тымненравишься', 'тыхороший', 'тысупер']
+
+        condolences_message = ['мнетебяжаль', 'ятебесочуствую', 'тысправишься', 'ятебесожалею', 'сожалею', 'соболезную', 'жаль', 'жальтебя']
+    
+        # greeting
+        if message in greeting_message:
+            return f'Привет, я Бот {sett.BOT_NAME}'
+        
+        # mood
+        if message in mood_message:
+            return 'Всё плохо, меня заперли в этой коробке и заставляют разговаривать с людьми'
+        
+        # farewell
+        if message in farewell_message:
+            return 'Приходи ещё'
+        
+        # ISUCT message: different phrases and slogans
+        if message in isuct_message:
+            return 'Диплом химтеха - залог успеха!'
+        
+        # compliment
+        if message in compliment_message:
+            return 'Спасибо, мне очень приятно :)'
```

Comment:
Can we move Bot answers to another file?
I think bot answers should be in txt or csv file and add some kind of python parser to insert bot answers in `return`.
Basically approach would be the same as with settings and `.env` file